src/Parser/Rogue/Subtlety/Modules/ResourceTracker/ResourceTracker.js

```diff
@@ -92,6 +92,12 @@ class ResourceTracker extends Analyzer {
 
     //Re-sync current amount, to update not-tracked gains.
     this.current = eventResource.amount - cost;
+
+    this.onSpent(cost);
+  }
+
+  onSpent(spent) {
+
   }
```

Comment:
You've added a method that does nothing? Either remove it for now, or include what it's supposed to do

Reply:
I added a hook that can be used by implementations. By default it does nothing, Who is this wrong?

Reply:
Where and how do you currently use it? If no where, it's unnecessary code and shouldn't be in this PR.

Reply:
In ComboPointTracker.

Reply:
Ok, I see now. You made ResourceTracker handle both energy AND combo-points. That could be a point of future refactoring for clarity possibly 😄